Packages/Tracking/Core/Runtime/Scripts/ActionDetectors/PinchDetector.cs

```diff
@@ -55,9 +55,45 @@ public enum PinchableFingerType
         public bool IsPinching => IsDoingAction;
 
         // Accessor actions for readability
-        public Action<Hand> OnPinchStart => onActionStart;
-        public Action<Hand> OnPinchEnd => onActionEnd;
-        public Action<Hand> OnPinching => onAction;
+        public Action<Hand> OnPinchStart
+        {
+            get
+            {
+                return onActionStart;
+            }
+
+            set
+            {
+                onActionStart = value;
+            }
+        }
+
+        public Action<Hand> OnPinchEnd
+        {
+            get
+            {
+                return onActionEnd;
+            }
+
+            set
+            {
+                onActionEnd = value;
+            }
+        }
+
+        public Action<Hand> OnPinching
+        {
+            get
+            {
+                return onAction;
+            }
+
+            set
+            {
+                onAction = value;
+            }
+
+        }
```

Comment:
Likewise this could be altered:

```suggestion
        public Action<Hand> OnPinchStart
        {
            get => onActionStart;
            set => onActionStart = value;
        }

        public Action<Hand> OnPinchEnd
        {
            get => onActionEnd;
            set => onActionEnd = value;
        }

        public Action<Hand> OnPinching
        {
            get => onAction;
            set => onAction = value;
        }
```

Reply:
Done